File: frontend/src/app/shared/dto/user/address-request.ts
export class AddressRequest {

  private _street: string;
  private _state: string;
  private _zip: number;
  private _city: string;


  get street(): string {
    return this._street;
  }

  set street(value: string) {
    this._street = value;
  }

  get state(): string {
    return this._state;
  }

  set state(value: string) {
    this._state = value;
  }

  get zip(): number {
    return this._zip;
  }

  set zip(value: number) {
    this._zip = value;
  }

  get city(): string {
    return this._city;
  }

  set city(value: string) {
    this._city = value;
  }
}

File: src/main/java/com/mediheroes/mediheroes/dto/user/AddressRequest.java
package com.mediheroes.mediheroes.dto.user;

import javax.validation.constraints.NotNull;

public class AddressRequest {

    @NotNull
    private String city;

    @NotNull
    private String state;

    @NotNull
    private String street;

    @NotNull
    private Integer zip;

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public Integer getZip() {
        return zip;
    }

    public void setZip(Integer zip) {
        this.zip = zip;
    }
}
